packages/docs/components/fpo.svg

```diff
@@ -0,0 +1,5 @@
+<svg>
```

Comment:
This is used as an FPO placeholder and will be removed once all final assets are in place.

Reply:
Should this be moved to public assets?